pkg/s3select/sql/utils.go

```diff
@@ -50,7 +50,7 @@ func (e *JSONPathElement) String() string {
 	return ""
 }
 
-// Removes double quotes in quoted identifiers
+// String removes double quotes in quoted identifiers
```

Comment:
Can we not fix the staticcheck updates in another PR instead of this change? - since its a rewrite with a large change, it looks quite intrusive as its touching unexpected parts.

Reply:
Too late now.